src/makefile.all

```diff
@@ -285,16 +285,16 @@ LIB_ARGS = $(OBJPATH)ar.arg
 
 TARGETS = $(STAT_LIB)
 
-all: $(C_ARGS) $(PKT_STUB) $(OBJPATH)cflags.h $(TARGETS)
+all: $(PKT_STUB) $(OBJPATH)cflags.h $(TARGETS)
 	@echo All done
 
 $(STAT_LIB): $(OBJS) $(LIB_ARGS)
 	$(AR) $@ @$(LIB_ARGS)
 
-$(OBJPATH)%.o: %.c
+$(OBJPATH)%.o: %.c $(C_ARGS)
```

Comment:
GNU-make has the syntax `$(OBJPATH)%.o: %.c | $(C_ARGS)` called
*Order-only prerequisites*. Section 4.2 of the GNU-make [manual](https://www.watt-32.net/misc/make.pdf).

Reply:
I think order-only prerequisites are only meant for directories.  In this case `$(C_ARGS)` has the makefile as prerequisite and you generally want to rebuild everything if the makefile changes.